src/nvim/quickfix.c

```diff
@@ -200,9 +200,9 @@ qf_init_ext (
   char_u          *pattern;
   char_u          *fmtstr = NULL;
   int col = 0;
-  char_u use_viscol = FALSE;
-  int type = 0;
-  int valid;
+  bool use_viscol = false;
+  char_u type = 0;
+  char_u valid;
```

Comment:
`type` and `idx` (below) are `char_u`s, but `valid` is a `bool` or I'm missing somelthing.